lib/Gfx/TUM_Draw.c

```diff
@@ -774,6 +766,9 @@ int vBindDrawing(void) // Should be called from the Drawing Thread
 		goto err_make_current;
 	}
 
+    if(renderer)
```

Comment:
@PhilippvK 

This popped up with valgrind. Been so long since I looked into how SDL uses renderers, does this look correct?

Reply:
I can have another look at it. The problem is, that there is almost no example code on the web, where thoses handles are cleaned up completely.